LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright 2021 RStudio, PBC
+Copyright 2023 Posit Software, PBC
```

Comment:
Interesting, I didn't know you could use a LICENSE file like this for MIT; I thought CRAN required their template.

Reply:
They do, so `renv` creates a CRAN-compliant LICENSE file in R CMD build: https://github.com/rstudio/renv/blob/release/1.0.0/R/license.R

This lets `renv` have a "real" LICENSE file on GitHub, but have a CRAN version for package submissions.